### PR TITLE
feat: add npy reader

### PR DESCRIPTION
**Pull request by WenheLI**
This PR adds support for reading npy file.

### CODE REVIEW

**Review by steveoni — APPROVED** [review]
LGTM!

File: src/generic/npy.ts [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+    name: "int64",
+    size: 64,
+    arrayConstructor: BigInt64Array
+  },
+  "<f4": {
+    name: "float32",
+    size: 32,
+    arrayConstructor: Float32Array
+  },
+  "<f8": {
```
Comment (yorkie) [inline src/generic/npy.ts:74]: Shall we keep a consistent style for quotes?

**Comment by yorkie** [comment]
I'm not sure if we should use the name `npy`, it follows Python's convention, our user might not familiar with that :)

**Comment by WenheLI** [comment]
> I'm not sure if we should use the name `npy`, it follows Python's convention, our user might not familiar with that :)

`npy` should be fine, but we do need to mention it in our documentation.

**Comment by WenheLI** [comment]
@steveoni @risenW - Gentle ping for review.

**Review by risenW — APPROVED** [review]
LGTM

**Pull request merged**